modules/client/src/client-common/utils.ts

```diff
@@ -76,3 +75,10 @@ export function isProposalId(propoosalId: string): boolean {
   const regex = new RegExp(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,}$/i);
   return regex.test(propoosalId);
 }
+
+export function isIpfsCid(hash: string): boolean {
```

Comment:
there are 2 different versions of CID. 
https://docs.ipfs.tech/concepts/content-addressing/#cid-versions

why not using a library that supports everything https://github.com/multiformats/js-multiformats

Reply:
Preventing bloatware is an asset, if this library is lightweight then +1 here

BTW, this will need to change as soon as the metadata is a URI

Reply:
ok then please implement both versions so that the test is at least correct.

Reply:
then we are good. maybe a comment would help, that we support v0 and v1 in case v2 comes out

Reply:
Sure, I'll add a comment :+1:

Reply:
given the other PR in progress, I would repurpose this function into something like `isIpfsUri()`, more generic. This will allow detecting potential different formats if we need in the future